MonoGame.Framework/Platform/SDL/SDLGamePlatform.cs

```diff
@@ -233,6 +238,37 @@ private void SdlRunLoop()
                                 break;
                         }
                         break;
+
+                    case Sdl.EventType.DropFile:
+                        if (ev.Drop.WindowId != _view.Id)
+                            break;
+
+                        if (!_view.IsFileDropHandled)
```

Comment:
DropFile event isn't something that is gonna happen every input action or something like that, it shouldn't be necessary to protect against it if nothing is hooked up, its only gonna create problems if you try to add/remove stuff from it.